AdyenDropIn/Presentation/DropInNavigationController.swift

```diff
@@ -43,7 +43,11 @@ internal final class DropInNavigationController: UINavigationController {
     }
     
     internal func present(asModal component: PresentableComponent) {
-        pushViewController(wrapInModalController(component: component, isRoot: false), animated: true)
+        if component.requiresModalPresentation {
+            pushViewController(wrapInModalController(component: component, isRoot: false), animated: true)
+        } else {
+            present(component.viewController, animated: true, completion: nil)
+        }
```

Comment:
since `BrowserComponent` uses the `PresentationDelegate`, does this mean merchants have to check the `requiresModalPresentation` as well when implementing the `PresentationDelegate `?

Reply:
yes, it is happening in `IntegrationExampleComponents`